src/components/ArticleList.jsx

```diff
@@ -0,0 +1,88 @@
+/** @format */
+
+import { fetchAllArticles } from "../api/api";
+import { useState } from "react";
+import { useEffect } from "react";
+import ArticleCard from "./ArticleCard";
+import { FaRegThumbsDown, FaRegThumbsUp } from "react-icons/fa";
+
+function ArticleList() {
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
+		fetchAllArticles().then((res) => {
+			const data = res.data.articles;
+			setArticles(data);
+		});
+	}, [setArticles]);
+
```

Comment:
Would be good to see a loading state here

Reply:
The free tier of render can be shockingly slow at times - without a loading spinner etc it can even look like your webpage doesn't work